city_scrapers/spiders/example.py

```diff
@@ -0,0 +1,231 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class StlAldermenSpider(CityScrapersSpider):
+    name = "stl_public_service"
+    agency = "St. Louis Board of Public Service"
+    timezone = "America/Chicago"
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+    start_urls = [
+        (
+            "https://www.stlouis-mo.gov/government/departments/public-service/index.cfm"
+        )
+    ]
+
+    def __init__(self, *args, **kwargs):
+        self.agenda_map = defaultdict(list)
+        super().__init__(*args, **kwargs)
+
+    def parse(self, response):
+        self._parse_links(response)
+        yield from self._parse_meetings_page(response)
+
+    def _parse_meetings_page(self, response):
+        urls = [
+            (
+                "https://www.stlouis-mo.gov/events/"
+                "past-meetings.cfm?span=-30&department=332"
```

Comment:
The department number for the Board of Public service is `209`. So change the part of the url path to `&department=209`.